src/satosa/routing.py

```diff
@@ -136,17 +139,20 @@ def endpoint_routing(self, context):
         :return: registered endpoint and bound parameters
         """
         if context.path is None:
-            satosa_logging(logger, logging.DEBUG, "Context did not contain a path!", context.state)
+            logger.debug("Context did not contain a path!")         
```

Comment:
There is some extra whitespace at the end of the line here.

Some calls are still missing the session id.